www/photos.html

```diff
@@ -4,125 +4,138 @@
-                    <button class="btn btn-info" data-dest="delete-no">No</button>
-                </div>
+    </div>
+    <div class="row">
+        <div class="col-md-12">
+            <div class="btn-group" id="delete-confirm2">
+                <button class="btn btn-default">Really sure?</button>
+                <button class="btn btn-danger" data-dest="delete-final">Yes</button>
+                <button class="btn btn-info" data-dest="delete-no">No</button>
             </div>
         </div>
-        <div id="photos">
-
+    </div>
+    <div class="row">
+        <div class="col-md-9">
+            <nav id="page-nav" aria-label="Photo page selector">
+                <ul class="pagination pagination-sm">
+                    <li class="page-item move-backwards disabled" data-amount="all">
+                        <a class="page-link" href="javascript:void(0);" data-toggle="tooltip" data-placement="top"
+                           title="Back to start">|&lt;</a>
+                    </li>
+                    <li class="page-item move-backwards disabled" data-amount="10">
+                        <a class="page-link" href="javascript:void(0);" data-toggle="tooltip" data-placement="top"
+                           title="Back 10 pages">&lt;&lt;</a>
+                    </li>
+                    <li class="page-item move-backwards disabled" data-amount="1">
+                        <a class="page-link" href="javascript:void(0);" data-toggle="tooltip" data-placement="top"
+                           title="Back 1 page">&lt</a>
+                    </li>
+                    <li class="page-item page-number disabled">
+                        <a class="page-link" href="javascript:void(0);">1</a>
+                    </li>
+                    <li class="page-item move-forwards disabled" id="next" data-amount="1">
+                        <a class="page-link" href="javascript:void(0);" data-toggle="tooltip" data-placement="top"
+                           title="Forward 1 page">&gt;</a>
+                    </li>
+                    <li class="page-item move-forwards disabled" data-amount="10">
+                        <a class="page-link" href="javascript:void(0);" data-toggle="tooltip" data-placement="top"
+                           title="Forward 10 pages">&gt;&gt;</a>
+                    </li>
+                    <li class="page-item move-forwards disabled" data-amount="all">
+                        <a class="page-link" href="javascript:void(0);" data-toggle="tooltip" data-placement="top"
+                           title="Forward to end">&gt;|</a>
+                    </li>
+                </ul>
+            </nav>
+        </div>
+        <div class="col-md-3">
+            <nav id="page-size-nav" aria-label="Page size selector" class="float-right">
+                <ul class="pagination pagination-sm">
+                    <li class="page-item disabled"><span class="page-link">Size:</span></li>
```

Comment:
Instead of size, maybe a better wording would be "Photos per page: "

Reply:
When I tried putting large words in here it broke the default styling and I wasn't really keen on trying to fix that :)

I'll look into another way of getting a label attached to this block and get back to you